lib/PhasarLLVM/TypeHierarchy/DIBasedTypeHierarchy.cpp

```diff
@@ -167,11 +184,24 @@ DIBasedTypeHierarchy::DIBasedTypeHierarchy(const LLVMProjectIRDB &IRDB) {
 
     // -- Filter all struct- or class types
 
+    // NOLINTNEXTLINE(cppcoreguidelines-avoid-c-arrays,modernize-avoid-c-arrays)
+    static constexpr llvm::dwarf::Tag DwarfTags[] = {
+        llvm::dwarf::DW_TAG_class_type,
+        llvm::dwarf::DW_TAG_structure_type,
+        llvm::dwarf::DW_TAG_union_type,
+    };
```

Comment:
does it really have to be an array? what about `static constexpr auto DwarfTags = {`?

Reply:
If using `auto`, the type deduces to a `std::initializer_list` and I am unsure, whether it is safe to use this way (e.g., lifetimes, compiler-dependent bahavior, ...).

To be safe, I chose to use an array here.